net/strongswan/patches/010-gcc14.patch

```diff
@@ -0,0 +1,22 @@
+--- a/src/libcharon/plugins/uci/uci_parser.c
```

Comment:
Can we get a descriptive header?

Reply:
Still hoping to merge this, but I need the review comments resolved.

Reply:
Ping @neheb